org.eclipse.wildwebdeveloper.tests/pom.xml

```diff
@@ -31,6 +31,7 @@
 				<artifactId>tycho-surefire-plugin</artifactId>
 				<version>${tycho-version}</version>
 				<configuration>
+					<argLine>${tycho.testArgLine} ${ui.test.vmargs}</argLine>
```

Comment:
This is making build fail on Linux, see automated CI review.
`ui.test.vmargs` must be set as empty by default, and overriden in some mac-specific profile

Reply:
Done